### DISCUSSION TITLE
Updating to LinkToAnotherRecord field [May 25]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2234] [Discord Threads]
> 🙋‍♂️ **Canx**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### - You can double click 'colored bubble' (or)   
#### - Use expand icon on right hand side of that cell (becomes visible on click)  
#  
> 🙋‍♂️ **Canx**  
#### I found it! I must change the column to "Primary value". I was confused because in spanish was translated as "primary key", that it's a different thing...  
#  
> 🙋‍♂️ **Canx**  
#### At the end of this document is explained: https://github.com/nocodb/nocodb/blob/19d5f0114bbb9438896b2e6a97843e406ca030da/packages/noco-docs/content/en/setup-and-usages/primary-value.md  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/978701605870387300)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
